{"uuid": "56ef7d9c-8b6a-483a-87e4-76598f911ce1", "children": ["3dc167f7-da57-4743-bc04-a549f4437c6e", "238b1233-1150-40f6-a7bb-aac002ced40d"], "befores": [{"name": "initialize_browser_state", "status": "broken", "statusDetails": {"message": "AttributeError: 'Settings' object has no attribute 'get_base_url'\n", "trace": "  File \"D:\\GitHub\\autotests-ui\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"D:\\GitHub\\autotests-ui\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"D:\\GitHub\\autotests-ui\\venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"D:\\GitHub\\autotests-ui\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\GitHub\\autotests-ui\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\GitHub\\autotests-ui\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 930, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\GitHub\\autotests-ui\\fixtures\\browsers.py\", line 12, in initialize_browser_state\n    context = browser.new_context(base_url=settings.get_base_url())\n                                           ^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\GitHub\\autotests-ui\\venv\\Lib\\site-packages\\pydantic\\main.py\", line 1026, in __getattr__\n    raise AttributeError(f'{type(self).__name__!r} object has no attribute {item!r}')\n"}, "start": 1761567574321, "stop": 1761567576023}], "start": 1761567574321, "stop": 1761567576210}